crates/kubelet/src/image/mod.rs

```diff
@@ -0,0 +1,3 @@
+#![allow(missing_docs)]
+
+pub mod client;
```

Comment:
This is probably too nested. I think bringing up the client into here is probably sufficient